### ISSUE TITLE
Cannot read property 'organization' of undefined when using environment variables

### ISSUE THREAD

**Issue by actuallymentor** [expo/sentry-expo#103]
I'm trying to use `sentry-expo/upload-sourcemaps` with environment variables. As per [documentation](https://docs.expo.io/versions/latest/guides/using-sentry/)  I'm passing the following things into the environment:

```
organization = SENTRY_ORG
project = SENTRY_PROJECT
authToken = SENTRY_AUTH_TOKEN
(optional) url = SENTRY_URL
```

When building expo I get:

```
Running postPublish hook: sentry-expo/upload-sourcemaps
Error uploading sourcemaps to Sentry: Cannot read property 'organization' of undefined
Verify that your Sentry configuration in app.json is correct and refer to https://docs.expo.io/versions/latest/guides/using-sentry.html
```

I did not specify a `config` key into the sentry config in the `app.json`. When I supply an empty config (thinking the env variables might override it) I receive an error as well.

**Comment by actuallymentor** [comment]
Issue was not fixed in #104 

At versions:

```json
"dependencies": {
    "@sentry/browser": "^5.15.5",
    "sentry-expo": "^2.0.3"
}
```

With environment:

```shell
organization='xxx'
project='xxx'
authToken='xxxxxxxxxxx'
```

I still receive:

```shell
Running postPublish hook: sentry-expo/upload-sourcemaps
Error uploading sourcemaps to Sentry: Cannot read property 'organization' of undefined
Verify that your Sentry configuration in app.json is correct and refer to https://docs.expo.io/versions/latest/guides/using-sentry.html
Publish complete
```

**Comment by cruzach** [comment]
It was fixed, just hadn't been released yet. Just deployed the fix in `sentry-expo@2.0.4`
